README.md

```diff
@@ -28,6 +25,12 @@ is a tool for testing a CSAF Trusted Provider according to [Section 7 of the CSA
 ### [csaf_aggregator](docs/csaf_aggregator.md)
 is a CSAF Aggregator, to list or mirror providers.
 
+## Other stuff
+
+### [examples](./examples/README.md)
+are small examples how to use `github.com/csaf-poc/csaf_distribution` as an API.
+Currently this work in progress. They may be extended and/or changed in the future.
```

Comment:
Currently this is work in progress. As they are subject to change they may be extended/altered in the future.